src/plugin.py

```diff
@@ -452,6 +452,22 @@ def _get_not_added_free_games(owned_games):
         log.info("Plugin shutdown.")
         await self.authentication_client.shutdown()
 
+    async def get_game_time(self, game_id, context):
+        if self.local_client.load_local_files_done:
+            for game in self.local_client.config_parser.games:
+                try:
+                    blizzard_game = Blizzard[game.uid]
+                except KeyError:
+                    log.warning(f'[{game.uid}] is not known blizzard game. Skipping')
+                    return None
+
+                if blizzard_game.id == game_id:
+                    last_played = int(game.last_played)
```

Comment:
game.last_played may be None. This should be covered.

Reply:
@bartok765 OK